quality/pre-push-recommended.sh

```diff
@@ -20,9 +20,9 @@
         detekt \
         verifyWebViewCount \
         assembleAmazonWebViewDebugAndroidTest \
-        testAmazonWebViewDebug
+        lintAmazonWebViewDebug \
+        testAmazonWebViewDebug \
+        connectedAmazonWebViewDebugAndroidTest
```

Comment:
I'm all for running UI tests before merging, but this seems kinda annoying.  It means you will only be able to push without no-verify if you have 1 device connected (no more, no fewer), and only after a long wait.

I've been running UI tests before each merge, but before each push seems excessive.